ui/apps/platform/src/Containers/Integrations/utils/integrationUtils.ts

```diff
@@ -53,6 +53,16 @@ export function clearStoredCredentials<I extends IntegrationBase>(
     return integration;
 }
 
+export function getEditDisabledMessage(type) {
+    if (type === 'clusterInitBundle') {
+        return 'This Cluster Init Bundle can not be edited. Create a new Cluster Init Bundle or delete an existing one';
```

Comment:
* **This** sounds like a contrast with that other which can be edited.
* **Create** or **delete** is an unusual contrast. Is the intended meaning to delete an existing one if it has become obsolete after you create a new one? If yes, ask the docs team to suggest a clear and concise statement. Probably as separate sentences. If docs do not already have a starting point for the wording, then it is important to fill the gap there too.

Reply:
That language is already in the response. I just moved it out to the tooltip. I'm not opposed to new language, but I'd rather that be a follow-on task, to prevent this Occam's Razor solution from expanding into a long discussion.